tock/hours/tests/test_views.py

```diff
@@ -1012,3 +1028,11 @@ def test_ReportingPeriodDetailView_current_employee_toggle(self):
             len(response.html.find_all('tbody')), 3
         )
         self.former_employee
+
+class TestFloatViewIntegration(TestCase):
+    fixtures = FIXTURES
+    def setUp(self):
+        self.userdata = UserData.objects.get(pk=1)
+
+    def test_basic(self):
+        pass
```

Comment:
Hehe! Looks like that whole class may have been a vestige of something earlier. Will get it cleaned up.